notebooks/end-to-end-structured/labs/5a_train_keras_vertex_babyweight.ipynb

```diff
@@ -0,0 +1,743 @@
+{
```

Comment:
<p>Add back a mention "Lab Task" or "Exercise" to help locate the exercises and add a brief description of what's asked.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Done. Thanks.</p>